staging/src/k8s.io/apiextensions-apiserver/examples/client-go/pkg/client/clientset/versioned/typed/cr/v1/example.go

```diff
@@ -46,6 +46,7 @@ type ExampleInterface interface {
 	List(ctx context.Context, opts metav1.ListOptions) (*v1.ExampleList, error)
 	Watch(ctx context.Context, opts metav1.ListOptions) (watch.Interface, error)
 	Patch(ctx context.Context, name string, pt types.PatchType, data []byte, opts metav1.PatchOptions, subresources ...string) (result *v1.Example, err error)
+
```

Comment:
Can you undo whatever is causing this blank line to cut down on the diff size?

Reply:
Apologies, I thought I had the fix for that merged in. Adding.